jigna/qt_server.py

```diff
@@ -26,6 +26,9 @@
 
 #: Path to jigna.js file
 JIGNA_JS_FILE = join(abspath(dirname(__file__)), 'js', 'dist', 'jigna.js')
+JIGNA_VUE_JS_FILE = join(
```

Comment:
No, there are two separate files `jigna.js` is the ng one and `jigna-vue.js` is the vue one.